shared/src/test/diff/codegen/Nested.mls

```diff
@@ -1262,9 +1262,6 @@ fun main =
     else g(x - 1)
   let g(x: Int): Int = f(x)
   f
-//│ ╔══[ERROR] identifier `g` not found
```

Comment:
Why did this error disappear?

Reply:
This error is from `PreTyper`, not from `Typer`. I have muted the output of `PreTyper` in all tests, as `PreTyper` is not yet fully implemented. Currently, `PreTyper` can only complete the symbol resolution required by UCS.

By using the global flag `:ShowPreTyperErrors`, you can turn on the error output of `PreTyper`. I have already copied the use cases that will cause `PreTyper` to fail to `pretyper/Errors.mls` for future research when implementing `PreTyper`.